apiserver/facades/client/client/client_test.go

```diff
@@ -62,6 +62,9 @@ func (s *serverSuite) SetUpTest(c *gc.C) {
 		"authorized-keys": coretesting.FakeAuthKeys,
 	}
 	s.baseSuite.SetUpTest(c)
+	s.PatchValue(client.GetCachedModel, func(ctx facade.Context, modelUUID string) (client.ModelCache, error) {
```

Comment:
`baseSuite` embedds `JujuConnSuite`, which sets up a cache and cache worker. If none of the tests in the suite hit the cache, we can ignore it. I don't think we have to patch anything out.

Reply:
removed as part of work from the next one.